lib/hanami/web/rack_logger.rb

```diff
@@ -57,8 +57,7 @@ def log_request(env, status, elapsed)
           ip: env[HTTP_X_FORWARDED_FOR] || env[REMOTE_ADDR],
           path: env[SCRIPT_NAME] + env[PATH_INFO].to_s,
           length: extract_content_length(env),
-          params: env[ROUTER_PARAMS],
-          time: Time.now,
```

Comment:
@solnic Why? This `time` argument was passed to Rack. I can't find the Rack counterpart at this moment.

How are we measuring elapsed time? Via `dry-monitor`?

Reply:
@jodosha this is just a log timestamp, it's generated on the logger side so no need to instantiate another time instance here

Reply:
@jodosha oh and elapsed time is passed from dry-monitor, yes (last argument in `log_request`)